OpenOversight/app/utils/constants.py

```diff
@@ -1,13 +1,71 @@
 import os
 
 
+# Data Constants
+US_STATE_ABBREVIATIONS = [
```

Comment:
We have that `us` package that should provide these abbreviations

Reply:
Done!